.github/workflows/proxysql_integration_tests.yml

```
@@ -162,6 +169,140 @@ jobs:
+        run: go build -o dbdeployer .
+
+      - name: Download and unpack PXC
+        run: |
+          TARBALL="Percona-XtraDB-Cluster_${PXC_VERSION}_Linux.x86_64.glibc2.17-minimal.tar.gz"
+          URL="https://downloads.percona.com/downloads/Percona-XtraDB-Cluster-LATEST/Percona-XtraDB-Cluster-8.0.27/binary/tarball/$TARBALL"
+          mkdir -p "$SANDBOX_BINARY"
+          curl -L -f -o "/tmp/$TARBALL" "$URL"
+          ./dbdeployer unpack "/tmp/$TARBALL" --sandbox-binary="$SANDBOX_BINARY"
+
+      - name: Test PXC with ProxySQL
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2012:info:1:11: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)